test/unit/line-ripple/index.test.tsx

```diff
@@ -3,10 +3,16 @@ import * as td from 'testdouble';
 import {assert} from 'chai';
 import {shallow} from 'enzyme';
 import LineRipple from '../../../packages/line-ripple/index';
+import {MDCLineRippleAdapter} from '@material/line-ripple/adapter';
 import {coerceForTesting} from '../helpers/types';
 
 suite('LineRipple');
 
+function getAdapter(foundation: unknown): MDCLineRippleAdapter {
```

Comment:
RE
Argument of type 'import("/home/travis/build/material-components/material-components-web-react/packages/line-ripple/node_modules/@material/line-ripple/foundation").MDCLineRippleFoundation' is not assignable to parameter of type 'import("/home/travis/build/material-components/material-components-web-react/node_modules/@material/line-ripple/foundation").MDCLineRippleFoundation'.

Looks like imports for testing and for the component are different. Do you know how to fix this error?


It sounds like one of your deps are out of sync with each other. There are the node_modules for the @material/line-ripple package and the tests. Try deleting the node_modules directory of @material/line-ripple and the root directory, and running `npm i` again.

Reply:
the error is from Travis, it does not have any cache issues I think

Reply:
What if you do:
```

function getAdapter(instance: LineRipple): MDCLineRippleAdapter {
  // @ts-ignore
  return instance.foundation_.adapter_;
}```

Reply:
yes, that works, thanks